aries_cloudagent/connections/models/diddoc_v2/tests/test_diddoc.py

```diff
@@ -598,6 +598,138 @@ async def test_universal_resolver(self):
         result3 = DIDDoc.deserialize(universal_resolver_DID_3)
         assert result3.service[0].id
 
+        # Same key reference in multiple fields
+        universal_resolver_DID_4 = {
```

Comment:
Each of these should be individual tests. If they're grouped as shown here, if one fails after changes to schema validation or similar, the entire test would fail short-circuiting the rest of the checks. Only after fixing the first issue would you see that there are more issues that were hidden because the test failed early.